sdk/keyvault/azure-keyvault-certificates/pom.xml

```diff
@@ -11,8 +11,8 @@
   </parent>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-keyvault-certificates</artifactId>
-  <version>4.0.0-preview.5</version> <!-- {x-version-update;com.azure:azure-keyvault-certificates;current} -->
+  <artifactId>azure-security-keyvault-certificates</artifactId>
+  <version>4.0.0-preview.5</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
```

Comment:
I just merged https://github.com/Azure/azure-sdk-for-java/pull/5860

The versions have changed and it will cause merge conflicts. Please resolve them before merging this.

Reply:
reverted to P5 as discussed